src/buffer/out/search.cpp

```diff
@@ -144,3 +144,8 @@ ptrdiff_t Search::CurrentMatch() const noexcept
 {
     return _index;
 }
+
+bool Search::IsOk() const noexcept

```

Comment:
```suggestion
bool Search::IsSearchInvalid() const noexcept
```
maybe? Just something a bit more specific would be nice imo, because `search->IsOk()` is kinda ambiguous out-of-context.